build.gradle.kts

```diff
@@ -14,12 +14,13 @@ allOpen {
 dependencies {
   implementation("com.github.kittinunf.result:result:4.0.0")
   implementation("com.github.kittinunf.result:result-coroutines:4.0.0")
-  implementation("com.beust:klaxon:5.4")
-  implementation("com.amazonaws:aws-java-sdk-s3:1.11.959")
-  implementation("io.sentry:sentry-spring-boot-starter:4.2.0")
-  implementation("net.javacrumbs.shedlock:shedlock-spring:4.20.0")
-  implementation("net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.20.0")
+  implementation("com.beust:klaxon:5.5")
+  implementation("com.amazonaws:aws-java-sdk-s3:1.11.973")
+  implementation("io.sentry:sentry-spring-boot-starter:4.3.0")
+  implementation("net.javacrumbs.shedlock:shedlock-spring:4.21.0")
+  implementation("net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.21.0")
   implementation("nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect:2.5.2")
+  implementation("org.apache.xmlgraphics:batik-all:1.14")
```

Comment:
What you have is fine and will work.  Would be helpful to add this perhaps so we know why for this particular dependency:

```
  constraints {
    implementation("org.apache.xmlgraphics:batik-all:1.14") {
      because("previous version 1.13 pulled from Apache POI 5.0.0 has CVE")
    }
  }
```